lib/dry/configurable.rb

```diff
@@ -133,5 +135,11 @@ def create_config_for_nested_configurations
     def nested_configs
       _settings.select { |setting| setting.value.kind_of?(::Dry::Configurable::NestedConfig) }.map(&:value)
     end
+
+    # @private
+    def raise_already_defined_config(key)
+      raise AlreadyDefinedConfig,
+        "Cannot add setting `#{key}`, #{self} is already configured"
```

Comment:
Could we update this to be more in line with https://github.com/dry-rb/dry-configurable/blob/master/lib/dry/configurable/config.rb#L73 please? Or vice versa, not bothered which.

i.e. `Cannot add setting +#{key}+, #{self} is already configured`